meson.build

```diff
@@ -0,0 +1,76 @@
+project('libebml', 'cpp',
+  version : '1.4.2',
+  default_options : ['warning_level=1', 'cpp_std=c++14'],
+)
+
+ebml_conf = configure_file(output: 'ebml_export.h', configuration: {'EBML_DLL_API': ''})
```

Comment:
What is the Meson equivalent of `CXX_VISIBILITY_PRESET` in CMake ? This is a feature that caused problems in the past and needs to be handled properly.

Reply:
In meson, you set the kwarg `gnu_symbol_visibility: 'hidden'` on the target (and make the configured ebml_export.h set EBML_DLL_API to something useful for exporting symbols, rather than just trying to squelch compiler warnings by setting it to an empty string).

Reply:
In the current ebml_export.h file, EBML_DLL_API is not defined.